test/index.test.js

```diff
@@ -174,11 +174,12 @@ describe('Index Tests', async () => {
       __ow_path: 'list-everything',
       limit: 3,
       service: 'Wrong Service',
+      format: 'access.log',
     });
     assert.equal(typeof result, 'object');
+    assert.equal(result.body.results.length, 3);
     assert.ok(Array.isArray(result.body.results));
     assert.ok(!result.body.truncated);
-    assert.equal(result.body.results.length, 3);
     assert.deepEqual(result.headers, {
       'content-type': 'application/json',
```

Comment:
I might misunderstand the use case, but if you request something as an access.log, shouldn't the response be a simple text (with corresponding mime type) rather than a JSON array?

Reply:
Right, but rendering doesn't respect the formatting for some reason! Which is why i just resorted to an array it seems to respect that every object deserves it's own line when displayed?

Reply:
I don't understand....why wouldn't it respect the formatting?

Reply:
You might just have to join the array to get a text body.

Reply:
I still think this should be a text result.

Reply:
ah, now I see: yes, just add `.join('\n')`